src/app/rpmostree-builtin-kargs.c

```diff
@@ -52,12 +52,13 @@ static GOptionEntry option_entries[] = {
 };
 
 static GVariant *
-get_kargs_option_variant (void)
+get_kargs_option_variant (gboolean is_authorized_prior)
 {
   GVariantDict dict;
 
   g_variant_dict_init (&dict, NULL);
   g_variant_dict_insert (&dict, "reboot", "b", opt_reboot);
+  g_variant_dict_insert (&dict, "authorized", "b", is_authorized_prior);
```

Comment:
We can't do that - that entirely subverts the authorization check, as any caller could set that to `TRUE`.  What we should be using here is polkit's "imply" bits.  See [the docs](https://www.freedesktop.org/software/polkit/docs/latest/polkit.8.html).  And for example (on my workstatation under `/usr/share/polkit-1/actions`):

```
  <action id="org.fedoraproject.FirewallD1.all">
    <description>Firewall</description>
    <message>System policy prevents inspecting and changing firewall</message>
    <defaults>
      <allow_any>auth_admin_keep</allow_any>
      <allow_inactive>auth_admin_keep</allow_inactive>
      <allow_active>auth_admin_keep</allow_active>
    </defaults>
    <annotate key="org.freedesktop.policykit.imply">org.fedoraproject.FirewallD1.info org.fedoraproject.FirewallD1.config org.fedoraproject.FirewallD1.config.info org.fedoraproject.FirewallD1.direct org.fedoraproject.FirewallD1.direct.info org.fedoraproject.FirewallD1.policies org.fedoraproject.FirewallD1.policies.info</annotate>
  </action>

        <action id="org.freedesktop.login1.inhibit-block-shutdown">
                <description>Allow applications to inhibit system shutdown</description>
...
                <annotate key="org.freedesktop.policykit.imply">org.freedesktop.login1.inhibit-delay-sleep</annotate>
```

etc.

Reply:
Ah ha, that is what I was looking for :), thanks for the info, will do the change when I am back online.